### ISSUE TITLE
Chapter outline font isn't legible on Story Maps

### ISSUE THREAD

**Issue by CourtneyLee333** [techmatters/terraso-product#1046]
## Description
The text for the chapter outline is too low in contrast to meet accessibility guidelines or be legible. It should be changed to #358EAE.
